src/mprismetadata.cpp

```diff
@@ -19,6 +17,210 @@
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::trackId
+    \brief Identifies the track inside a playlist
+
+    Identifier for the current track, should be a valid DBus object path.
+    Maps to MPRIS property 'mpris:trackid'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::duration
+    \brief Length of the media in milliseconds
+
+    Maps top MPRIS property 'mpris.length'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::coverArtUrlSmall
+    \brief Small version of a cover image for the media
+
+    An url for a cover image of the media.
+    This field is only provided for QtMultimedia compatibility, the
+    value from coverArtUrlLarge is preferred.
+
+    Maps to MPRIS property 'mpris:artUrl'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::coverArtUrlLarge
+    \brief Large version of a cover image for the media
+
+    An url for a large cover image of the media. If defined, will
+    override coverArtUrlSmall.
+
+    Maps to MPRIS property 'mpris:artUrl'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::albumTitle
+    \brief Title of the album of the media
+
+    The name of an album the media is a part of, or undefined
+    if not applicable or known.
+    Maps to MPRIS property 'xesam:album'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::albumArtist
+    \brief Artist of the album of the media
+
+    The artist of an album the media is part of.
+    Maps to MPRIS property 'xesam:albumArtist'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::contributingArtist
+    \brief Artist or artists who contributed the media
+
+    The artist or artists who have made the media.
+    Eg. band, DJ, singer, director.
+    Maps to MPRIS property 'xesam:artist'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::lyrics
+    \brief Lyrics of the media
+
+    Maps to MPRIS property 'xesam:asText'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::audioBpm
+    \brief Beats per minute of audio
+
+    Maps to MPRIS property 'xesam:audioBPM'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::autoRating
+    \brief An automatic rating for the media
+
+    A rating for the media decided via automated means in the range [0, 1].
+
+    Maps to MPRIS property 'xesam:autoRating'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::comment
+    \brief A list of free form comments about the media.
+
+    Maps to MPRIS property 'xesam:comment'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::composer
+    \brief List of composers of the media.
+
+    Maps to MPRIS property 'xesam:composer'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::year
+    \brief The year the media was produced.
+
+    Convenience accessor for the year part of the date property.
+    \sa MprisMetaData::date
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::date
+    \brief The date the media was produced.
+
+    Maps to MPRIS property 'xesam:contentCreated'.
+*/
+
+/*!
+    \qmlproperty var MprisMetaData::chapterNumber
+    \brief
+
+    Maps to MPRIS property 'xesam:discNumber'
```

Comment:
Chapter -> Disc, huh?

Reply:
I'd suggest renaming the property to discNumber.

Reply:
Done.